Source/Core/Core/PowerPC/JitArm32/JitArm_FloatingPoint.cpp

```diff
@@ -534,3 +536,48 @@ void JitArm::frsqrtex(UGeckoInstruction inst)
 	gpr.Unlock(fpscrReg, rA);
 }
 
+void JitArm::fmsubx(UGeckoInstruction inst)
+{
+	INSTRUCTION_START
+	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
+
+	u32 a = inst.FA, b = inst.FB, c = inst.FC, d = inst.FD;
+
+	ARMReg vA0 = fpr.R0(a);
+	ARMReg vB0 = fpr.R0(b);
+	ARMReg vC0 = fpr.R0(c);
+	ARMReg vD0 = fpr.R0(d, false);
+
+	ARMReg V0 = fpr.GetReg();
+
+	VMUL(V0, vA0, vC0);
+
+	VSUB(vD0, V0, vB0);
+
+	fpr.Unlock(V0);
+}
+
+void JitArm::fnmsubx(UGeckoInstruction inst)
+{
+	INSTRUCTION_START
+	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
+
+	u32 a = inst.FA, b = inst.FB, c = inst.FC, d = inst.FD;
+
+	ARMReg vA0 = fpr.R0(a);
+	ARMReg vB0 = fpr.R0(b);
+	ARMReg vC0 = fpr.R0(c);
+	ARMReg vD0 = fpr.R0(d, false);
+
+	ARMReg V0 = fpr.GetReg();
+
+	VMUL(V0, vA0, vC0);
```

Comment:
I /think/ the ARM fmsub instructions act differently than the PowerPC instructions if I remember correctly.